test/tests/vectorpostprocessors/parallel_consistency/parallel_consistency.i

```diff
@@ -0,0 +1,79 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+
+  # To make this deterministic
+  [Partitioner]
+    type = GridPartitioner
+    nx = 2
+    ny = 1
+    nz = 1
+  []
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+[]
+
+[BCs]
+  [left]
+    type = DirichletBC
+    variable = u
+    boundary = 'left'
+    value = 0
+  []
+  [right]
+    type = DirichletBC
+    variable = u
+    boundary = 'right'
+    value = 1
+  []
+[]
+
+[VectorPostprocessors]
+  [constant]
+    type = ConstantVectorPostprocessor
+    value = '3 4'
+    execute_on = 'TIMESTEP_BEGIN'
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+[]
+
+[Outputs]
+  execute_on = 'timestep_end'
+  csv = true
```

Comment:
Are you outputting csv in this test? I don't see anything that would. I'd prefer to see this removed if you really aren't. The reason is that this has caused us weird race condition issues when people refactor, copy tests around and output multiple file formats at the same time without explicitly setting prereqs. It has bitten us enough that I'd prefer to keep it cleaned up.

Reply:
👍